src/System.Private.ServiceModel/src/System/ServiceModel/Description/TypeLoader.cs

```diff
@@ -1369,6 +1372,12 @@ private static MessagePartDescription CreateParameterPartDescription(XmlName def
                                                                               i,
                                                                               -1));
                 }
+                else if (memberInfo.IsDefined(typeof(MessagePropertyAttribute), false))
+                {
+                    messageDescription.Properties.Add(CreateMessagePropertyDescription(memberInfo,
```

Comment:
Formatting nit, can you make this cleaner. E.g. if you line break after Add(, you could make everything be on two lines.
```csharp
    messageDescription.Properties.Add(
        CreateMessagePropertyDescription(memberInfo, new XmlName(memberInfo.Name), i));
```